search-services/1.3/using/index.md

```diff
@@ -289,13 +289,14 @@ Single terms, phrases, and so on can be combined using "`NOT`" in upper, lower,
 These queries search for nodes that contain the terms `yellow` in any content.
```

Comment:
Update to say ""These queries search for nodes that contain the term "yellow" but do not contain "banana" in any content."

Reply:
Updated.